### ISSUE TITLE
The plugin is not working (update currency value in DB) in PHP 7.3

### ISSUE THREAD

**Issue by RempadDesign** [responsiv/currency-plugin#33]
Hi. We have developed an application and the plugin is not working (updating the DB value for the currency hourly..daily etc.) if we set the PHP version to 7.3. If we set this up to 7.2 it works just fine.

**Comment by daftspunk** [comment]
Can you provide more detail, what error do you see when updating the DB value for the currency hourly..daily etc? Can you refer to the code that is failing? Thanks

**Comment by RempadDesign** [comment]
> Can you provide more detail, what error do you see when updating the DB value for the currency hourly..daily etc? Can you refer to the code that is failing? Thanks

Hi, we didn't realy dig in this very much. We just saw that on the test server we had the php 7.2 version and the values from responsiv_currency_exchange_rates were added every 1 hour as it should. When we migrated the API to the clients server, the PHP version was 7.3 and since the migration no values were added no more to the table. After we changed the PHP version to 7.2 and checke the extensions (of the PHP) to be the same, the values in the responsiv_currency_exchange_rates table were again populated.

Maybe it's a matter of extensions...i don't really know the cause because no error was thrown, just the new values not been added to responsiv_currency_exchange_rates.

Cheers.
